### PR TITLE
Cop 12427 event at border

### PR DESCRIPTION
**Pull request by michaelruocco**
implements calls to display event at border form when one has been raised upon submissions of target frontline action

### CODE REVIEW

File: client/src/pages/targets/forms/cop-record-outcome.json [hunk 1]
```diff
@@ -38,10 +38,6 @@
               {
                 "label": "Insufficient Resources",
                 "value": "INSUFFICIENT_RESOURCES"
-              },
-              {
-                "label": "Target Withdrawn",
-                "value": "WITHDRAWN"
```
Comment (michaelruocco) [inline client/src/pages/targets/forms/cop-record-outcome.json:44]: this is not related to E@B but needed to be removed as it is no longer applicable and was also broken! it raised a border event if this option was used due to a bug in the BPMN, which is incorrect!

File: client/src/pages/targets/v2/RecordOutcome.test.jsx [hunk 2]
```diff
@@ -152,4 +152,34 @@ describe('RecordOutcome', () => {
...
+
+  it('should show border event form if border event raised when front line action recorded', async () => {
+    const reactNaviMock = jest.requireMock('react-navi');
+    const navigationUrls = [];
+    reactNaviMock.useNavigation = () => {
+      return {
+        navigate: (url) => navigationUrls.push(url),
+      };
+    };
+    localStorage.setItem(LS_KEYS.VIEW, VIEW.AIRPAX);
```
Comment (charles474) [inline client/src/pages/targets/v2/RecordOutcome.test.jsx:164]: For the `VIEW` constant, the options are now 

```
export const VIEW = {
  AIRPAX_IO: 'AIRPAX_IO',
  AIRPAX_HO: 'AIRPAX_HO',
  AIRPAX_RCCU: 'AIRPAX_RCCU',
  NONE: 'RORO',
  RORO: 'RORO',
  RORO_V2: 'RORO_V2',
};
```

`VIEW.AIRPAX_IO` Should do the trick.

**Review by charles474 — APPROVED** [review]
LGTM

File: client/src/pages/targets/v2/RecordOutcome.test.jsx (hunk 2 above)
Comment (michaelruocco) [reply client/src/pages/targets/v2/RecordOutcome.test.jsx:164]: Cool, will update now, thanks!

Comment (michaelruocco) [reply client/src/pages/targets/v2/RecordOutcome.test.jsx]: done

**Pull request merged**